http/src/main/resources/reference.conf

```diff
@@ -254,7 +254,7 @@ azure {
      # All app versions excluded up until the switch to WSM-controlled KubernetesNamespace
      # https://github.com/DataBiosphere/leonardo/pull/3666
      chart-versions-to-exclude-from-updates = [
-       "0.2.345",
+       "0.2.346",
```

Comment:
This seems like a bug -- the exclusion shouldn't be auto-bumped

Reply:
Looks like this should actually be `0.2.341` (see [original commit](https://github.com/DataBiosphere/leonardo/commit/54423c09a8a0db43b08956f12928dcd8d89b59d9#diff-9024d8d3f171d065fa3b61dbc7dc2c56bdc6587644fde470722fdc92e7068f61R257) and [autobump](https://github.com/DataBiosphere/leonardo/commit/ae82662ee0a79ce7768d90ae8d709ae7dc861d52#diff-9024d8d3f171d065fa3b61dbc7dc2c56bdc6587644fde470722fdc92e7068f61R257)) and  and was previously auto-bumped. It seems to do this if the current version is included in the excluded versions... the bandaid would be "avoid doing that" but I'm not sure if there's a better long-term solution without some pattern-matching akin to changes in [this PR](https://github.com/broadinstitute/terra-helmfile/pull/4483)

Reply:
Thanks for fixing! I also changed it back to `0.2.341` in [this PR](https://github.com/DataBiosphere/leonardo/pull/3710)